templates/app.html

```diff
@@ -40,7 +40,6 @@
   <!-- Styles -->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.3/css/materialize.min.css">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <link href="static/css/dist/app.min.css" rel="stylesheet">
 
   <!-- <link rel="stylesheet" href="{{ url_for('static', filename='css/dist/bundle.css') }}"> -->
   <link rel="stylesheet" href="http://127.0.0.1:8080/static/css/dist/bundle.css">
```

Comment:
When running in prod, this wouldn't work, but I'm not worried about that right now. I still need to implement filename hashing and change some other stuff. You can see though that right above it I had commented out a line to import the built bundle file instead.